src/Ojs/SiteBundle/Resources/views/Journal/journal_index.html.twig

```diff
@@ -343,6 +343,9 @@
                                                         <i class="fa fa-caret-right"></i>
                                                         <a href="{{ article.publicURI }}">{{ article.title }}</a>
 
+                                                        <br>
+                                                            {{ "first_page"|trans }} {{ article.firstPage }} - {{ "last_page"|trans }} {{ article.lastPage }}
```

Comment:
First and last pages can be empty. Are you sure the page won't look bad this way?